src/style.css

```diff
@@ -1 +1,76 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
```

Comment:
Ensure that the font URL `/src/fonts/Avenir-Heavy.ttf` is correct and accessible. Consider using a relative path consistently, such as `./fonts/Avenir-Heavy.ttf`, to avoid potential issues.